src/common/sckaddr.cpp

```diff
@@ -849,7 +849,14 @@ bool wxIPV4address::Hostname(unsigned long addr)
         return false;
     }
 
-    m_origHostname = Hostname();
+    const wxUint8
+        addr1 = (addr >> 24) & 0xFF,
+        addr2 = (addr >> 16) & 0xFF,
+        addr3 = (addr >>  8) & 0xFF,
+        addr4 = (addr >>  0) & 0xFF;
+
+    m_origHostname = wxString::Format("%hhu.%hhu.%hhu.%hhu", addr1, addr2, addr3, addr4);
```

Comment:
To be honest I'm not sure what's the advantage of using `wxUint8` above and `%hhu` here compared to just using `unsigned` and `%u`, is there something I'm missing?

Reply:
No, there's no advantage; just wanted to be explicit.